rabbitmq/rabbitmq.go

```diff
@@ -352,16 +419,20 @@ func (client *DefaultClient) publishToLndhubExchange(ctx context.Context, invoic
 		},
 	)
 	if err != nil {
-		captureErr(client.logger, err)
 		return err
 	}
 
-	client.logger.Debugf("Successfully published invoice to rabbitmq with RHash %s", invoice.RHash)
+	client.logger.Infoj(log.JSON{
+		"subroutine":   "invoice publisher",
+		"message":      "succesfully published invoice",
+		"payment_hash": invoice.RHash,
+	})
```

Comment:
is some more metadata helpful like the key (and exchange?) 
the key is computed it might be interesting to add.